### PR TITLE
fix link to custom metrics flow diagram

### PR DESCRIPTION
**Pull request by Otterpohl**
## Description

Troubleshooting document contains broken img link.

## Type of change

_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Changelog entry

_Please put a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note
Fixes the link to custom metrics flow diagram in the troubleshooting guide
```

### CODE REVIEW

File: Documentation/troubleshooting.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ A common problem related to `ServiceMonitor` identification by Prometheus is rel
 
 <!-- do not change this link without verifying that the image will display correctly on https://prometheus-operator.dev -->
```
Comment (simonpasquier) [inline Documentation/troubleshooting.md:52]: this comment says it all :)

Comment (Otterpohl) [reply Documentation/troubleshooting.md]: Apologies, didn't quite understand it until digging into the site a bit further! I have updated the comments link to make it abundantly clear.
Any way to independently fix the github doc link?

Comment (simonpasquier) [reply Documentation/troubleshooting.md]: > Any way to independently fix the github doc link?

If you mean fixing the link so it works properly in GitHub, the answer is no (at least I have no solution).

Comment (Otterpohl) [reply Documentation/troubleshooting.md]: fair enough!

**Pull request closed without merging**